### DISCUSSION TITLE
Brightness & Volume issues

### DISCUSSION THREAD

**Discussion by Bighead255** [MonitorControl/MonitorControl#1521] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

A couple of days ago without updating MonitorControl or my mac my volume & brightness adjustments started pairing up (i.e. whenever I changed it, it would increase/decrease by 2) and the mute button no longer works. I updated MonitorControl to 4.2.0 but still having the same problem. But the monitor now also makes the whites super bright when the brightness is pushed above half. None of these problems existed a week ago. Any ideas to solve this? Thanks.

### Expected behavior

Brightness and volume adjustments to work as expected.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.5 Monterey
- Mac model: Mac Studio (2022)
- MonitorControl version: 4.2.0
- Monitor(s): Dell U2723QE
- Monitor Cable(s)/Connection(s): USB-C
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
You might have the app running somehow in 2 instances or an other app of similar kind is running at the same time. Try rebooting and if it does not work look at Activity Monitor to check out what is happening.

**Comment by Bighead255** [comment]
Just had a check and there's only 1 instance running according to Activity Monitor. I also restarted my computer but made no difference.

**Reply by waydabber** [reply]
I see. You might still have an other app (BetterDisplay, Lunar, flux, BTT etc) installed?

If not, then uninstall the app and please check whether [BetterDisplay](https://betterdisplay.pro) works.

**Reply by Bighead255** [reply]
Hi there, I tried all of the above and updated my mac to the latest version (OS 14.3.1) but still to no effect.
